controllers/provider-azure/charts/internal/cloud-provider-config/templates/cloud-provider-config.tpl

```diff
@@ -6,6 +6,9 @@ routeTableName: "{{ .Values.routeTableName }}"
 securityGroupName: "{{ .Values.securityGroupName }}"
 subnetName: "{{ .Values.subnetName }}"
 vnetName: "{{ .Values.vnetName }}"
+{{- if hasKey .Values "vnetResourceGroup" }}
```

Comment:
Can you add this to the `values.yaml` as well?

Reply:
I did not add the `vnetResourceGroup` key:value pair to the `values.yaml`, because deploying in an existing vnet is not the default.

Reply:
Ahh now I got it. Yes sure will add this for documentation purposes. Thank you!

Reply:
done